EIPS/eip-6963.md

```diff
@@ -0,0 +1,230 @@
+---
+eip: 6963
+title: Multi Injected Provider Discovery
```

Comment:
I feel like it doesn't roll out of the tongue so much if we make it longer so if you are ok I will leave it as is

Reply:
How about "Wallet detection without injection"?

Reply:
I do agree here somewhat - although it doesnt roll off the tounge as @pedrouid noted it does provide clarity on what kind of "provider" 
```suggestion
title: Multi Injected Wallet Provider Discovery
```

Reply:
@danfinlay EIP-6963 still requires injection but just uses events instead of globals

@Adamj1232 yeah I guess we could "Wallet" if everyone finds it helpful

Reply:
"Event-Based Wallet Discovery" perhaps